opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -373,6 +373,7 @@ def __init__(
         out: typing.IO = sys.stdout,
         formatter: typing.Callable[[Span], str] = lambda span: span.to_json()
         + os.linesep,
+        **kwargs
```

Comment:
This change will be required else the code in init_tracing method 
```provider.add_span_processor(
            BatchExportSpanProcessor(exporter_class(**exporter_args))
        ) ```
fails with error "unexpected argument" service name.

Reply:
got it! thanks.

Reply:
thank you!

Reply:
Shouldn't we only add `service_name` as an argument?

Reply:
I could add a default parameter service_name=None.  I thought of adding **kwargs because in future if additional parameters get passed, we might have to come back and change again.
I could still go ahead and make the change suggested. Please let me know.

Reply:
**kwargs is more robust but this `service_name` requirement is restriction placed on the constructors by an external component (auto-instrumentation), which is not ideal. If changes happen in the future if we want to pass additional configurable parameters via auto-instrumentation, all of these parameters must be added to the separate exporters that currently are supported (zipkin, jaeger, etc.), so this would have to be addressed regardless. For now, it is better to keep these parameters explicit so we know which ones are available and are needed to exist for this auto-instrumentation feature.

Reply:
@lzchen ok got it.  Will make the changes.

Reply:
@lzchen - The tests seems to fail with an error **"GitHub Actions has encountered an internal error when running your job.
"**. I ran all the tests locally (except pypy3)  and they were passing.
Any pointers to understand whether its my failure or a general failure would be helpful.